docs/storage_change_status.md

```diff
@@ -0,0 +1,128 @@
+# Storage change status
+
+> Analysis of possible storage change statuses in [EIP-2200] / [EIP-1283].
+
+## Cost constants
+
+| Constant | Value |
+| -------- | ----- |
+| SLOAD    | 200   |
+| SET      | 20000 |
+| RESET    | 5000  |
+| CLEARS   | 15000 |
+
+
+## Storage change statuses
+
+- `0` - a zero storage value,
+- `X` - non-zero storage value,
+- `Y` - non-zero storage value different from `X`,
+- `Z` - non-zero storage value different form `X` and `Y`,
+- `...` - any value.
+
+
+<table>
```

Comment:
Why not use a markdown table?

This one can format it for you: https://www.tablesgenerator.com/markdown_tables

Reply:
No rowspans.

Reply:
Never mind, found the way to "display file" so that it renders.